src/interaction.wiki

```diff
@@ -848,12 +880,20 @@ are made for.
 
 >>
 
+It looks great, but if we refresh the page after creating an account,
+we can see there is a problem. We are creating the same account over and 
+over again. This because we are calling the create_account_service each time
+we refresh the page, obviously we don't want that. To solve this issue we will
```

Comment:
It would be more readable with a full stop before "obviously". Also, future tense is to be avoided in technical writing. ("We implement ...")